README.md

```diff
@@ -30,11 +30,12 @@ Load an existing Excel file:
 ```
 $ tiron example.xlsx
 ```
-
+-   `Arrow Keys` to navigate cells
 -   `e` to edit a cell and enter the value or formula.
 -   `q` to quit and save
 -   `+` to add a sheet
 -   `s` to go to the next sheet
+-   `a` to go to the previous sheet
```

Comment:
Seems like it would be good to call out these keyboard shortcuts.

Reply:
Awesome!